src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -620,7 +620,7 @@ VirtualRepeatController.prototype.containerUpdated = function() {
         this.repeatListExpression,
         angular.bind(this, function(items) {
           if (items && items.length) {
-            this.$$rAF(angular.bind(this, this.readItemSize_));
+            this.readItemSize_();
```

Comment:
I think there's a misunderstanding on what `requestAnimationFrame` does. You use it to register for a callback before the browser repaints the window. This is more relating to syncing graphical output with the renderer. You would use this to ensure you don't redraw the same frame before the browser has displayed it. Or, you can use it to make sure to make sure you don't animate a frame too fast.

The interval at which this is fired is completely browser dependent. One browser might do this at 60fps. Another might do 30fps. There are cases when the browser won't even render a frame, for example, when a tab/window is minimized.

I would look at the initial reasoning as to why rAF was implemented in the first place. It sounds like it was to "wait one frame", but that can be done with mdUtil.nextTick(). That's actually what $$rAF does if the browser doesn't support `requestAnimationFrame`. It'll create an angular timeout running at 16.66ms (1/60th) of a frame.